### ISSUE TITLE
In landscape mode, getting to opponents deck is difficult

### ISSUE THREAD

**Issue by Artanixuk** [JulianEberius/track-o-bot-ios#9]
I've found its possible, but there is an indivisible boundary where the box can be dragged to get down to the opponent deck part when trying to track games. Once you know where you can drag its easily, but maybe have color differentiation or in landscape have your deck and opponent deck side by side. I dunno if there's enough room for this though.
![img_0764](https://cloud.githubusercontent.com/assets/4305505/16978192/b0b93bcc-4e52-11e6-9c14-1cc024a65ff1.PNG)

**Comment by JulianEberius** [comment]
Yeah, the layout is not that great on smaller devices in landscape. I will try to find a better one, e.g. , move the two picker views (rotating controllers) next to each other in landscape mode.
